sdcm/cluster.py

```diff
@@ -2743,8 +2743,7 @@ def _publish_event(subtype, severity, duration=None, error=None, full_traceback=
             return result
         except Exception as details:  # pylint: disable=broad-except
             event_severity_on_failure = Severity.ERROR
-            if warning_event_on_exception and type(details) in warning_event_on_exception or \
-                    Exception in warning_event_on_exception:
+            if warning_event_on_exception and isinstance(details, warning_event_on_exception):
```

Comment:
warning_event_on_exception is list, may include few exceptions. This is the reason why I used `type(details) in warning_event_on_exception`

Reply:
Signature: isinstance(obj, class_or_tuple, /)